modules/gke-hub/main.tf

```diff
@@ -70,6 +70,20 @@ resource "google_gke_hub_feature" "default" {
   }
 }
 
+resource "google_gke_hub_feature_membership" "servicemesh" {
+  provider   = google-beta
+  for_each   = var.features.servicemesh ? var.clusters : {}
+  project    = var.project_id
+  location   = "global"
```

Comment:
`location` should probably be specified by the use instead of having it hard-coded

Reply:
"global" is the only location that works here.